mdsuite/calculators/einstein_distinct_diffusion_coefficients.py

```diff
@@ -230,29 +230,26 @@ def _post_operation_processes(self, species: Union[str, tuple] = None):
             result = self._fit_einstein_curve([self.time, abs(self.msd_array)])
             properties = {"Property": self.database_group,
                           "Analysis": self.analysis_name,
-                          "Subject": "_".join(species),
+                          "Subject": [species[0], species[1]],
                           "data_range": self.data_range,
-                          'data': -1*result[0],
-                          'uncertainty': result[1]}
+                          'data': [{'x': -1*result[0], 'uncertainty': result[1]}]
+                          }
             self._update_properties_file(properties)
         else:
             result = self._fit_einstein_curve([self.time, self.msd_array])
             properties = {"Property": self.database_group,
                           "Analysis": self.analysis_name,
-                          "Subject": species,
+                          "Subject": [species[0], species[1]],
                           "data_range": self.data_range,
-                          'data': result[0],
-                          'uncertainty': result[1]}
+                          'data': [{'x': result[0], 'uncertainty': result[1]}]
+                          }
             self._update_properties_file(properties)
 
         # Update the plot if required
         if self.plot:
             plt.plot(np.array(self.time) * self.experiment.units['time'], self.msd_array, label=species)
             plt.show()
 
-        if self.save:
```

Comment:
I think, you should be able to change this to

```py
if self.save:
    properties = {"Property": self.database_group,
                  "Analysis": self.analysis_name,
                  "Subject": species,
                  "data_range": self.data_range,
                  'data': [{'x': x, 'y': y} for x, y in zip(self.time, self.msd_array)],
                  'information': "series"}
    self._update_properties_file(properties)
```
But I am still unsure about the species

Reply:
The tuple in the 'Subject' throws and error. It needs to be a list, unless you have changed something else.